bower.json

```diff
@@ -7,9 +7,15 @@
     "jquery.cookie": "~1.4.1",
     "bootstrap": "~3.3.1",
     "datatables-bootstrap3-plugin": "~0.2.0",
-    "angular": "~1.3.13",
+    "angular": "~1.3.15",
```

Comment:
Is there a reason why you changed the dependency for angular? Is 1.3.13 not supported anymore?